src/main/java/me/hydos/blaze4d/api/GlobalRenderSystem.java

```diff
@@ -130,25 +130,28 @@ public static Identifier generateId(int glId) {
      */
     public static void render() {
         Blaze4D.rosella.common.device.waitForIdle();
-
         Blaze4D.rosella.baseObjectManager.renderObjects.clear();
+
         for (ConsumerRenderObject renderObject : currentFrameObjects) {
-            Blaze4D.rosella.baseObjectManager.addObject(renderObject);
+            Blaze4D.rosella.common.fboManager.getPresentingFbo().objectManager.addObject(renderObject);
```

Comment:
do we really want an object manager per-fbo? is this really needed? i forgot exactly what restricted us to it. are pipelines fbo specific? if they can be changed with dynamic state pipelines we can worry about it later when we do graph stuff.

Reply:
they need to be because fbos are designed to allow rendering different "scenes" to different perspectives.

Reply:
pipelines are FBO specific iirc. At least the render target is for the pipeline